SPECS/xcp-clipboardd.spec

```diff
@@ -0,0 +1,36 @@
+Name:           xcp-clipboardd
+Version:        1.0.2
+Release:        1
```

Comment:
Add %{dist} (or `%{?dist}`, I haven't made up my mind. In the context of XCP-ng or any CentOS derivative, it doesn't matter).

=> `1{%dist}` will be expanded to `1.xcpng` at build time.